lib/evm/instruction/impl.ex

```diff
@@ -9,6 +9,7 @@ defmodule EVM.Instruction.Impl do
   alias EVM.SubState
```

Comment:
I was thinking of breaking this into `lib/evm/instruction/impl/{arithmetic, bitwise, etc...}.ex` to make this file a little more manageable, what do ya think?

Reply:
Cool, going to do this in a follow-on PR 👍